beacon_node/network/src/network_beacon_processor/sync_methods.rs

```diff
@@ -660,13 +660,23 @@ impl<T: BeaconChainTypes> NetworkBeaconProcessor<T> {
                         // The peer is faulty if they send blocks with bad roots.
                         Some(PeerAction::LowToleranceError)
                     }
-                    HistoricalBlockError::InvalidSignature
-                    | HistoricalBlockError::SignatureSet(_) => {
+                    HistoricalBlockError::InvalidSignature(e) => {
                         warn!(
                             error = ?e,
                             "Backfill batch processing error"
                         );
-                        // The peer is faulty if they bad signatures.
+                        // The peer is faulty if they send bad signatures.
+                        Some(PeerAction::LowToleranceError)
+                    }
+                    HistoricalBlockError::InvalidBlobsSignature(_)
+                    | HistoricalBlockError::InvalidDataColumnsSignature(_)
+                    | HistoricalBlockError::Unexpected(_)
+                    | HistoricalBlockError::AvailabilityCheckError(_) => {
+                        warn!(
```

Comment:
Only internal errors should be a warn, peers can send invalid objects and users can't do anything about it.